build/travis/job_macos/generateGitLog.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Ah, right, a simple `#!/bin/sh` won't do on any platform where /bin/sh is not bash.